menu.rb

```diff
@@ -0,0 +1,21 @@
+# How many menu items would the user like to see?
+puts "Welcome to the crazy random menu generator!"
+puts "How many menu items would you like to create?"
+menu_items = gets.chomp.to_i
```

Comment:
Things to think about for the future: What will happen if the user does not enter an integer? What happens if the user enters an integer whose value is larger than the # of items you have in your lists?